pkg/controller/stacks/install/installjob_test.go

```diff
@@ -251,6 +355,85 @@ status:
  conditionedStatus: {}
 `
 
+	stackRawNoControllerImage = `---
```

Comment:
Did you want to make these functions too?  I'm fine with the blobs of text if not. It's messy but status quo.

Reply:
I was thinking about it, but ended up not doing it because there was less to modify in this file. I will think about it a little more.

Reply:
I decided to do this; resolving.